tools/docker/tests/test_flows.sh

```diff
@@ -91,14 +91,12 @@ test_channel_selection() {
     return $check_error
 }
 
-test_ap_capability_info_reporting() {
-    #TODO: Implement
-    return 1
-}
-
-test_client_capability_info_reporting() {
-    #TODO: Implement
-    return 1
+test_ap_capability_query() {
```

Comment:
If you change the name, it should also be changed in the help text and in the ALL variable. But I suppose that wasn't intentional...

Reply:
The ALL variable is fine but I do in fact need to fix `usage()` and I'll do that now.